examples/index.html

```diff
@@ -47,9 +47,47 @@
             <h3>Directly access to <span class="text-normal text-italic">bpmn-visualization</span> live examples.</h3>
             <div>
                 <p>
-                    <span>Technical details about the implementation are available in the</span>
+                    <span>Technical details about the examples implementation are available in the</span>
                     <a href="https://github.com/process-analytics/bpmn-visualization-examples" target="_blank" rel="noopener noreferrer">bpmn-visualization-examples</a> repository.
                 </p>
+                <p>
+                    If you need BPMN diagram files for examples loading local files, you can use resources from
+                <ul>
+                <li>the BPMN Model Interchange Working Group <a href="https://www.omgwiki.org/bpmn-miwg">(BPMN MIWG)</a>, an
+                        initiative "<i>to support, facilitate, and promote the interchange of BPMN Models</i>".
+                        <ul>
+                            <li><a href="https://github.com/bpmn-miwg/bpmn-miwg-test-suite" target="_blank" rel="noopener noreferrer">bpmn-miwg-test-suite</a></li>
+                            <li><a href="https://github.com/bpmn-miwg/bpmn-miwg-demos" target="_blank" rel="noopener noreferrer">bpmn-miwg-demos</a></li>
+                        </ul>
+                    </li>
+                    <li>
+                        <a href="https://github.com/process-analytics/bpmn-visualization-examples/blob/master/bpmn-files/README.md">bpmn-visualization
+                            BPMN test diagrams</a></li>
+                </ul>
+                </p>
+            </div>
+        </section>
+
+        <section class="col-12 mt-2 mb-2">
+            <h2>Demos</h2>
+            <div class="columns">
+                <div class="col-4 p-2">
+                    <div class="card bg-dark p-2">
+                        <div class="card-image">
+                            <img src="static/img/preview_demo_load_and_navigation.png" class="img-responsive" alt="load and navigation diagram demo">
+                        </div>
+                        <div class="card-header">
+                            <div class="card-title h5">Load and Navigation demo</div>
+                            <div class="card-subtitle text-gray">show most of features provided by <i>bpmn-visualization</i> about
+                                BPMN rendering and diagram navigation when loading BPMN local files
+                            </div>
+                        </div>
+                        <div class="card-body"></div>
+                        <div class="card-footer">
+                            <a href="../demo/index.html" target="_blank" class="btn btn-primary">go to demo</a>
```

Comment:
I thought we keep only the last demo as example. But it is ok to keep like this for nom 🙂

Reply:
Keeping the last demo version is managed independently in #91 because it depends on #88 first.